anchor/src/main.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anchor;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(rename_all = "kebab-case", about = "The Anchor of Libra \u{2693}")]
```

Comment:
Generally good practice to avoid unicode on the CLI, not sure how much you care. Probably won't matter in the end though